files/usr/lib/lua/aredn/olsr.lua

```diff
@@ -51,6 +51,14 @@ function model.getOLSRRoutes()
   return routes['routes']
 end
 
+-- Get the RF IP address of a neighbor node
+function model.getRemRFIP(ip)
+  local url=string.format("http://%s/cgi-bin/api?status=ip", ip )
```

Comment:
I think this is a no-no.   We should have this info from OLSR locally on the node instead of generating more network traffic.